pkg/skuba/actions/cluster/init/manifests.go

```diff
@@ -24,6 +24,7 @@ const (
 ## Default        : ""
 ## ServiceRestart : crio
 #
-CRIO_OPTIONS=--pause-image={{.PauseImage}}{{if not .StrictCapDefaults}} --default-capabilities="CHOWN,DAC_OVERRIDE,FSETID,FOWNER,NET_RAW,SETGID,SETUID,SETPCAP,NET_BIND_SERVICE,SYS_CHROOT,KILL,MKNOD,AUDIT_WRITE,SETFCAP"{{end}}
+### BUG [ CRIO v1.18.0rc1 ] - string parsing issue based on comma separators
```

Comment:
I was told by @saschagrunert that the new parsing is how it is intended to be, so it means that if we keep the `not StrictCapDefault` feature (which we are supposed to, as it's bound to caasp4 lifetime), we'll need to update this anyway for old provisioned clusters.